torch/_inductor/graph.py

```diff
@@ -118,8 +118,7 @@ def may_get_constant_buffer_dtype(constant_buffer):
 
 
 def is_magic_method(op):
-    magic_ops = {method_to_operator(m) for m in magic_methods}
-    return op in magic_ops
+    return op in map(method_to_operator, magic_methods)
```

Comment:
Alternatively you could move the set creation into the global scope and then `is_magic_method` becomes `O(1)`.

Reply:
I could but unlike proposed change that is 100% win, creating set uses more RAM and depending on how often it's used, it may also be less efficient

Reply:
> depending on how often it's used, it may also be less efficient

This function is called for every "normal" aten op that gets lowered, and those aren't magic methods which is the worst case for the `map` implementation. 

I don't think it matters too much either way though so w/e

Reply:
> and those aren't magic methods which is the worst case for the map implementation.

I'm not saying that current implementation is optimal but it's 100% better than the implementation it's trying to replace ;)